### DISCUSSION TITLE
Updated Sharepoint Domain - Existing PNP Modern Search Web Parts Broken

### DISCUSSION THREAD

**Discussion by Aixsponsa** [microsoft-search/pnp-modern-search#3678] [General]
Hello,

We recently changed our sharepoint domain. This resulted in the employee directory we built out to not display any results.

I have gone through every step I can think of to try and resolve this issue including replicating the current setup. I basically added every web part again separately, configured the settings to be identical, and copied over my custom properties. Still no results.

When I add a search results web part without configuring, it DOES show some results, though the localpeopleresults seems to not show anything at all.

Should I chalk this up to be a transition period with the Sharepoint domain change? Possible that it might take longer to update everything on the back end perhaps?

Any advice would be greatly appreciated. Has anyone else gone through a domain change with existing PNP web parts?

Thank you.

**Comment by kasperbolarsen** [comment]
Hi, that sounds like a painful experience 😬. 
I am pretty sure that this is not a PnP modern search issue,  but most likely a issue related to localpeopleresults. 
You should be able to verify that using either a classic search page or SP Editor/ SharePoint search query tool.

**Reply by Aixsponsa** [reply]
You are correct, sorry for the naivety. Using the SharePoint Search Query Tool, LocalPeopleResults returns 0 results. Thank you for the suggestion.

**Comment by Aixsponsa** [comment]
I just read through Microsoft documentation on what to expect after a domain change and they do say peoples profiles could take up to 24 hours to show up. I will just have to be patient before I investigate further. Thanks again.
